bin/duplicate_list/prog2_safe.ml

```diff
@@ -0,0 +1,4 @@
+open Combinators
+(* had to remove rec here too *)
+let duplicate_list_gen (s : int) (x : int) : int list =
+  if sizecheck s then [] else x :: []
```

Comment:
Hmm, I need to think about if this is a safe repair and what the alternatives would be